### DISCUSSION TITLE
Default table sort of a column fails when using the app_tables option, if it is a linked column

### DISCUSSION THREAD

**Discussion by lewysigns** [anvilistas/tabulator#78] [General]
**Preface**

I was going to make a issue, but not sure if I should. So I brought it here. Also, the app_tables feature is amazing and speeds up my application considerably, so thank you!

**Is your feature request related to a problem? Please describe.**

Looking at the error and the traceback, below, it looks like _data_loader.py is making a new query on the table ascending by a given column. This issue occurs when the column is based off of a column of a linked column. 

Example below "order" is a linked column and "rate" is a column of table "order".

```
anvil.tables.TableError: Cannot sort by nonexistent column "order.rate"
called from anvil-services/anvil/tables/v2/_table.py:111
called from app/tabulator/Tabulator/_data_loader.py:228
called from app/tabulator/Tabulator/_data_loader.py:280
called from app/tabulator/Tabulator/_data_loader.py:280
```

**Describe the solution you'd like**
I would like sorters to work on linked columns.

**Describe alternatives you've considered**
I have attempted to make customer sorters, but it seams to fail because the default behavior of the app_table option is to re-query the table.

**Additional context**

**I am considering making a PR for this issue**
Yes. I would need to look at the code more to understand.

**Comment by lewysigns** [comment]
Just thought of a work around, to avoid user's receiving the error.

I can pass the ```"header_sort":False``` when defining the column and have a custom sort method above in a sort input location, similar to the tabulator example given in the ReadMe

**Reply by s-cork** [reply]
Yeap that sounds about right, let me know if doesn't work for you

**Reply by lewysigns** [reply]
Actually I wasn't able to sort this out.

I am not sure its possible to do this with an app_tables query either. reference: https://anvil.works/forum/t/sorting-a-data-table-by-a-subcolumn-of-a-linked-column/6101?u=anthonys

Before it worked (not using app_tables) because I could set the value of the cell when using a dictionary to set the table data and now I am using customer formatter's to retrieve the data and display linked data in the table.

**Reply by s-cork** [reply]
Yeap that's exactly the issue here!
Very annoying! Sorry about that.
I'll see where that feature request has got to internally
